Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -277,22 +336,29 @@ void MaxEnt::exec() {
   // Read input workspace
   MatrixWorkspace_const_sptr inWS = getProperty("InputWorkspace");
   // Number of spectra
-  size_t nSpec = inWS->getNumberHistograms();
+  size_t nHist = inWS->getNumberHistograms();
   // Number of data points - assumed to be constant between spectra or
   // this will throw an exception
   size_t npoints = inWS->blocksize() * resolutionFactor;
   // Number of X bins
   const size_t npointsX = inWS->isHistogramData() ? npoints + 1 : npoints;
+  // Linear adjustment of calculated data
+  MatrixWorkspace_const_sptr dataLinearAdj = getProperty("DataLinearAdj");
+  // Constant adjustment of calculated data
+  MatrixWorkspace_const_sptr dataConstAdj = getProperty("DataConstAdj");
+  // Add spectra in reconstruction if false
+  const bool sumSpectra = !getProperty("PerSpectrumImage");
```

Comment:
This line is very hard to read. Please keep variable names as close as possible to the property names to help the person reading the code in this case: const bool perSpectrumImage = getProperty("PerSpectrumImage");